components/Articles/Articles.js

```diff
@@ -1,14 +1,13 @@
-import Link from "next/link";
-import fetch from "node-fetch";
-import { useState, useEffect } from "react";
+import fetch from 'node-fetch';
```

Comment:
These lines got changed because of the `.prettierrc` file that tells prettier how to format files.

In that file we have `"singleQuote": true` so prettier automatically changes the double quotes to single quotes